cmd/getkeystone.go

```diff
@@ -18,16 +16,19 @@ func init() {
 var getKeystoneCmd = &cobra.Command{
 	Use:   "get-keystone",
 	Short: "Get Keystone related attributes",
-	Long: `Get Keystone related attributes`,
+	Long:  `Get Keystone related attributes`,
 	RunE: func(cmd *cobra.Command, args []string) error {
 		cfgMgr := GetCfg()
 		zap.L().Debug("Get keystone password")
-		password, err := cfgMgr.GetKeystonePassword(serviceName)
+		_, err := cfgMgr.GetKeystonePassword(serviceName)
 		if err != nil {
 			zap.L().Info("Error getting keystone password")
 			return err
 		}
-		fmt.Println(password)
+		// Do not print password
+		// It causes a security alert
+		// Clear-text logging of sensitive information
+		// fmt.Println(password)
```

Comment:
Why not just remove this line?

Reply:
That makes this `get-keystone` command a no-op :)

Reply:
It returns nil - what's the use of printing - and what does the function do when it doesn't even print? Restoring it - but I think you were right - no one is using get-keystone (not hagrid too) https://github.com/search?q=org%3Aplatform9+getKeystoneCmd&type=code